src/v2/components/Transactions/Detail/index.jsx

```diff
@@ -61,7 +85,25 @@ const TransactionDetail = () => {
     </li>
   );
 
-  const tabNav = ['Applications: 3', 'status: success', 'code/course'];
+  const programMap = _.reduce(
+    transaction.instructions,
+    (a, i) => {
+      if (!a[i.program_id]) {
+        a[i.program_id] = i.program_id;
+      }
+
+      return a;
+    },
+    {},
+  );
+
+  const programs = _.values(programMap);
+
+  const tabNav = [
+    `Applications: ${programs.length}`,
+    'status: success',
+    'code/course',
```

Comment:
"course" > "source"

Reply:
I was wondering what that was supposed to be!

Reply:
It's a lovely little anagram!